html/webappapis/dynamic-markup-insertion/opening-the-input-stream/bailout-order.window.js

```diff
@@ -0,0 +1,29 @@
+function openHasTakenEffect(doc) {
```

Comment:
From this filename, I would expect this test to be complete, but the rest of the checks are in other files.

Since the security origin check needs its own file, maybe the right thing is to move `assertOpenHasNotTakenEffect(document)` into a shared helper file, and also add references to that file to this one. Or, rename the other file something like "bailout-order-same-origin". (And this one "bailout-order-xml-and-sync-script"?)

I also believe we have similar tests for the custom elements thing and the active parser thing. Can we similarly rename or consolidate those?

Reply:
For custom elements it's more difficult to test, in fact I haven't quite found a way to test custom elements' interactions with URl. The catch is that the check would only be triggered if the custom element is created by the parser.